src/token/api/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 edition = "2021"
-name = "token"
+name = "token-api"
```

Comment:
Since we are renaming this crate, lets call it `is20-token` to make it more explicit when using this crate as a dependency.